OCAExplorer/src/components/Demo.tsx

```diff
@@ -90,24 +90,31 @@ const demoStates = {
   }]
 }
 
-export type DemoState = "NotRunning" | "RunningIntro" | "RunningBranding" | "RunningAll"
+export enum DemoState{
+    NotRunning,
```

Comment:
I would maybe rename this to `NotSeenDemo`

Reply:
I think renaming `NotRunning` to `PausedDemo` would make more sense. What do you think? Since all states other than SeenDemo involve an incomplete demo and paused demo is to avoid causing the demo to exclusively play from the start when an overlay is chosen.